x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/MockElasticInferenceServiceAuthorizationServer.java

```diff
@@ -23,15 +23,19 @@ public class MockElasticInferenceServiceAuthorizationServer implements TestRule
     private static final Logger logger = LogManager.getLogger(MockElasticInferenceServiceAuthorizationServer.class);
     private final MockWebServer webServer = new MockWebServer();
 
-    public static MockElasticInferenceServiceAuthorizationServer enabledWithSparseEmbeddingsAndChatCompletion() {
+    public static MockElasticInferenceServiceAuthorizationServer enabledWithRainbowSprinklesAndElser() {
         var server = new MockElasticInferenceServiceAuthorizationServer();
 
         String responseJson = """
             {
                 "models": [
                     {
-                      "model_name": "model-a",
-                      "task_types": ["embed/text/sparse", "chat"]
+                      "model_name": "rainbow-sprinkles",
+                      "task_types": ["chat"]
+                    },
+                    {
+                      "model_name": ".elser_model_2",
```

Comment:
EIS will expose `elser-v2`. Not sure it matters for this test though.
See: https://github.com/elastic/elasticsearch/pull/120981

Reply:
I'll update it 👍 the model ID here isn't actually being used but might as well try to align it for the future when we do use it.